frontend/javascripts/admin/dataset/dataset_upload_view.js

```diff
@@ -230,14 +239,18 @@ class DatasetUploadView extends React.Component<PropsWithFormAndRouter, State> {
         initialTeams: formValues.initialTeams.map(team => team.id),
       };
 
-      await reserveDatasetUpload(formValues.datastore.url, reserveUploadInformation);
+      const datastoreUrl = formValues.datastore.url;
```

Comment:
I changed this in another PR which should be merged soon. After pulling the master in, this needs to be turned into:

```suggestion
      const datastoreUrl = formValues.datastoreUrl;
```